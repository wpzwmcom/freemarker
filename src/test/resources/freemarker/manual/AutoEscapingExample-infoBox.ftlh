<#ftl output_format="HTML"><#-- Needed for the Eclipse plugin -->

<@infoBox "Foo & bar" />
<@infoBox "Foo <b>bar</b>"?no_esc />

<#macro infoBox message>
  <div class="infoBox">
    ${message}
  </div>
</#macro>