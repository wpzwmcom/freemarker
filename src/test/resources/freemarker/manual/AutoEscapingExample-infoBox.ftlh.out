
  <div class="infoBox">
    Foo &amp; bar
  </div>
  <div class="infoBox">
    Foo <b>bar</b>
  </div>

